packages/nextjs/services/database/bg-imports/0003_build_data.ts

```diff
@@ -0,0 +1,212 @@
+    console.log(`Fetched ${buildsData.length} builds from BuidlGuidl API`);
+
+    // Get existing users from the database to avoid errors when importing likes and builders relationships
+    const userRows = await db.query.users.findMany({ columns: { userAddress: true } });
+    const existingBuilders = new Set(userRows.map(u => u.userAddress));
+    console.log(`Found ${existingBuilders.size} existing builders in local database`);
+
+    // Track ignored addresses for logging purposes
+    const ignoredBuilders = new Set<string>();
+
+    let buildsImported = 0;
+    let buildRelationshipsImported = 0;
+    let likesImported = 0;
+
+    const transformedBuilds = buildsData.map(build => {
+      const buildUuid = convertFirebaseIdToUuid(build.id);
+      return {
+        ...build,
+        uuid: buildUuid,
+        coBuilderAddresses: Array.isArray(build.coBuilders) ? build.coBuilders : [],
+      };
+    });
+
+    // Upsert builds section
+    console.log(`Importing ${transformedBuilds.length} builds into database...`);
+    const buildQueries = transformedBuilds.map(build => {
+      buildsImported++;
+      return db
+        .insert(builds)
+        .values({
+          id: build.uuid,
+          name: build.name,
+          desc: build.desc || null,
+          demoUrl: build.demoUrl || null,
+          videoUrl: build.videoUrl || null,
+          imageUrl: transformImageUrl(build.image),
+          githubUrl: build.branch || null,
+          buildType: mapBuildType(build.type, build.id),
+          buildCategory: mapBuildCategory(build.id),
+          submittedTimestamp: build.submittedTimestamp ? new Date(build.submittedTimestamp) : new Date(),
+        })
+        .onConflictDoNothing();
+    });
+    await db.executeQueries(buildQueries as any);
+    console.log(`✅ Builds import complete (${buildsImported} builds processed)`);
+
+    // Database builders handling
+    console.log(`Importing build-builder relationships...`);
+    function getBuilderRow(buildId: string, address: string, isOwner: boolean) {
+      if (existingBuilders.has(address)) {
+        buildRelationshipsImported++;
+        return db.insert(buildBuilders).values({ buildId, userAddress: address, isOwner }).onConflictDoNothing();
+      }
+      ignoredBuilders.add(address);
+      return null;
+    }
+
+    // Upsert build-builders
+    const builderQueries = [];
+    for (const build of transformedBuilds) {
+      // Add the main builder
+      const ownerRow = getBuilderRow(build.uuid, build.builder, true);
+      if (ownerRow) builderQueries.push(ownerRow);
+
+      // Add all co-builders
+      for (const addr of build.coBuilderAddresses) {
+        const coBuilderRow = getBuilderRow(build.uuid, addr, false);
+        if (coBuilderRow) builderQueries.push(coBuilderRow);
+      }
+    }
+    await db.executeQueries(builderQueries as any);
+    console.log(`✅ Build-builder relationships import complete (${buildRelationshipsImported} relationships created)`);
```

Comment:
Ohh instead of maintaining a separate `buildRelationshipsImported` count, 

we could just do: `builderQueries.length` , same for `likeQueries`